tests/certification/bindings/alicloud/dingtalk/README.md

```diff
@@ -0,0 +1,20 @@
+# Alicloud DingTalk binding certification testing
+
+This project aims to test the Alicloud DingTalk binding component under various conditions.
+
+## Test Plain
+
+### Authentication tests
+
+### Functional tests
+
+- Create request:
```

Comment:
Please, refer to our docs for scope for certification tests: https://docs.dapr.io/operations/components/certification-lifecycle/#certification-tests

Also, see how other components are doing. Kafka pubsub is a good example to follow.